doc/introtext.rst

```diff
@@ -0,0 +1,48 @@
+===================
+ALARA Users' Guide:
+===================
+
+Analytic and Laplacian Adaptive Radioactivity Analysis
+======================================================
+--------------
+
+INTRODUCTION:
+=============
+
+ The primary purpose of ALARA is to calculate the induced activation caused by neutron irradiation throughout a nuclear system (including fission reactors, 
```

Comment:
For long lines, it is useful to force a "hard line wrap" with only about 60 characters per line.  This ensures that when there are future modifications, they are more localized and easier to review.

Reply:
The text editor you use may have features to help with that.